test/kubemark/start-kubemark.sh

```diff
@@ -207,6 +207,9 @@ function create-and-upload-hollow-node-image {
   CURR_DIR=`pwd`
   cd "${MAKE_DIR}"
   RETRIES=3
+  if [[ -z "${FULL_REGISTRY-}" ]]; then
```

Comment:
I guess you can replace this whole if statement with sth like FULL_REGISTRY="${FULL_REGISTRY:-$CONTAINER_REGISTRY/..}"

Reply:
done